eng/Packages.props

```diff
@@ -18,6 +18,7 @@
     <PackageReference Update="SourceLink.Create.CommandLine" Version="2.1.2" />
     <PackageReference Update="System.CodeDom" Version="4.4.0" />
     <PackageReference Update="System.Collections.Immutable" Version="5.0.0" />
+    <PackageReference Update="System.Configuration.ConfigurationManager" Version="4.7.0" />
```

Comment:
Do we own this? Also thinking we should try running this through RPS.

Reply:
Sorry this is my first time making a PR in MSBuild so I'm not familiar with the process. Mihai created an exp/* branch for me [here](https://github.com/dotnet/msbuild/tree/exp/minglunli/toolsetConfigurationNet5) and it triggered an VS insertion and that [passed](https://devdiv.visualstudio.com/DevDiv/_releaseProgress?_a=release-pipeline-progress&releaseId=973302). Is that RPS or do we need to run something else for it?

And I do think we own the package as shown [here](https://www.nuget.org/packages/System.Configuration.ConfigurationManager/4.7.0). The package is owned by Microsoft and dotnetframework.

Reply:
Great! That is what I was thinking of with RPS. It failed symbol check, but I'm assuming that was innocuous.

I should have clarified that I was wondering if MSBuild owns S.C.CM. I hit a problem at one point when MSBuild updated a package that Roslyn owned, and we had mismatching versions with them. I think that sort of problem would have been caught by RPS, so I think this is good? But I don't feel very confident about that.

Reply:
Following up to our convo this morning: I talked to @cdmihai and he's not sure about who owns this package exactly either, but believes it should be fine since it is not referenced in VS's config.corext.

Reply:
I found AssemblyVersions.tt, and it isn't referenced there either.